doc/keepalive.md

```diff
@@ -4,25 +4,27 @@ The keepalive ping is a way to check if a channel is currently working by sendin
 
 This guide documents the knobs within gRPC core to control the current behavior of the keepalive ping.
 
-The keepalive ping is controlled by two important channel arguments -
+The send-side keepalive ping in core is controlled by the following channel arguments -
 
 * **GRPC_ARG_KEEPALIVE_TIME_MS**
   * This channel argument controls the period (in milliseconds) after which a keepalive ping is sent on the transport.
 * **GRPC_ARG_KEEPALIVE_TIMEOUT_MS**
   * This channel argument controls the amount of time (in milliseconds) the sender of the keepalive ping waits for an acknowledgement. If it does not receive an acknowledgment within this time, it will close the connection.
-
-The above two channel arguments should be sufficient for most users, but the following arguments can also be useful in certain use cases.
-
+* **GRPC_ARG_HTTP2_MAX_PINGS_WITHOUT_DATA**
+  * This channel argument controls the maximum number of pings that can be sent when there is no data/header frame to be sent. GRPC Core will not continue sending pings if we run over the limit. Setting it to 0 allows sending pings without such a restriction. (Note that this is an unfortunate setting that does not agree with [A8-client-side-keepalive.md](https://github.com/grpc/proposal/blob/master/A8-client-side-keepalive.md). There should ideally be no such restriction on the keepalive ping and we plan to deprecate it in the future.)
```

Comment:
I think we should use consistent case for "gRPC Core". It is called "GRPC Core" in the second sentence here, and "gRPC core" in another spot.

Reply:
Done